public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -72,6 +72,7 @@ export const LayerControlPanel = memo(
     mapState,
     zoom,
     mapConfig,
+    inDashboardMode,
```

Comment:
nit: isReadOnlyMode ??

Reply:
IMO, `inDashboardMode` can be more readable in the current code base, since we only have normal mode and dashboard mode.

Reply:
Maps shouldn't know who is calling LayerControl Panel. We don't want to tie that knowledge in our variable name.

Reply:
Will update the name when have related refactor task